portugues_pt/colaborar.md

```diff
@@ -14,6 +14,8 @@ Contribuía para [obras de caridade](caridade/caridade.md)
 
 Compraria [componentes eletronicos](componentes/componentes.md)
 
+Revolucionava o sistema de [ensino](ensino/alternativa.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.